packages/react-components/react-theme/src/alias/lightColorPalette.ts

```diff
@@ -30,7 +31,8 @@ const personaColorPaletteTokens = personaSharedColorNames.reduce((acc, sharedCol
     [`colorPalette${color}BorderActive`]: personaSharedColors[sharedColor].primary,
   };
 
-  return { ...acc, ...sharedColorTokens };
+  Object.assign(acc, sharedColorTokens);
```

Comment:
I think these could all be simplified a tad bit more to

```ts
return Object.assign(acc, sharedColorTokens);
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#return_value